mne/minimum_norm/spatial_resolution.py

```diff
@@ -0,0 +1,389 @@
+# -*- coding: utf-8 -*-
+# Authors: Olaf Hauk <olaf.hauk@mrc-cbu.cam.ac.uk>
+#
+# License: BSD (3-clause)
+"""Compute resolution metrics from resolution matrix.
+
+Resolution metrics: localisation error, spatial extent, relative amplitude.
+Metrics can be computed for point-spread and cross-talk functions (PSFs/CTFs).
+"""
+import numpy as np
+
+from mne import SourceEstimate
+from mne.utils import logger, verbose
+
+
+@verbose
+def resolution_metrics(resmat, src, function, metric, threshold=0.5):
+    """Compute spatial resolution metrics for linear solvers.
+
+    Parameters
+    ----------
+    resmat : array of shape [n_orient * n_vertices, n_vertices]
```

Comment:
```suggestion
    resmat : array, shape (n_orient * n_vertices, n_vertices)
```

at least that how we used to do.